packages/lib-classifier/src/store/feedback/strategies/datavis/graph2drange/reducer.spec.js

```diff
@@ -1,90 +1,82 @@
 import { expect } from 'chai'
 import graph2dRangeReducer from './reducer'
 
-describe('feedback drawing graph2dRange reducer', function () {
-  const rule = {
-    hideSubjectViewer: false,
-    id: '1234',
-    strategy: 'graph2dRange',
-    successEnabled: true,
-    successMessage: 'Success!',
-    tolerance: '1',
-    width: '2',
-    x: '15'
-  }
+const ruleWithStrings = {
```

Comment:
We could make a feedback rules factory to make setup of this consistent in tests.